Dockerfile

```diff
@@ -1,6 +1,10 @@
-FROM quay.io/openshift/origin-cli:latest
+# current latest is point to 4.16.0
+FROM quay.io/openshift/origin-must-gather:latest
```

Comment:
i thought about using tag, but from what i understand, this new feature with time is just introduced in 4.16
and they wanna make it GA in 4.17
then we will need to update this when 4.17 is out which might be in a couple of month.
meanwhile we are having 2 more new component in the pipeline reaching GA in summer, so we will update in this repo in the new future as well.
would it be better to use tag at that time?

Reply:
No issues, Wen - I proposed it as an optional, since even before your changes there was `latest` used. Thus I'm completely fine with what you propose :slightly_smiling_face: